app/controllers/miq_policy_controller/alerts.rb

```diff
@@ -589,6 +590,9 @@ def alert_valid_record?(alert)
       unless value_less_than.to_i.between?(0, 100)
         add_flash(_(template_error % ["<", non, between]), :error)
       end
+      if value_less_than && value_greater_than && value_less_than >= value_greater_than
```

Comment:
I recommend to consider adding parenthesis here for the last expression (value_less_than >= value_greater_than) . operator precedence should work fine, but for readability imo it'll be easier.